projects/storefrontlib/src/cms-components/cart/cart-coupon/cart-coupon.component.ts

```diff
@@ -6,11 +6,16 @@ import {
   CartVoucherService,
   AuthService,
   OCC_USER_ID_ANONYMOUS,
+  CustomerCouponService,
+  CustomerCoupon,
+  CustomerCouponSearchResult,
+  FeatureConfigService,
 } from '@spartacus/core';
 import { Observable, combineLatest } from 'rxjs';
 import { Subscription } from 'rxjs/internal/Subscription';
 import { map, startWith, tap } from 'rxjs/operators';
 
+const MAX_CUSTOMER_COUPON_PAGE = 100;
```

Comment:
shouldn't it be a part of the class, so it can be replaceable by customers? we cannot really influence this value in any way (when extending this component for example)

Reply:
update to make it as an attribute of the class.